docs/user-guide/dev-environment.md

```diff
@@ -646,42 +667,42 @@ Loading a `cpe` module will do the following:
 - All HPE Cray PE modules will be updated so their default version is the
   one from the PE you have selected
 
-For example, if you have a code that uses the Gnu compiler environment, FFTW
-and NetCDF parallel libraries and you want to compile in the (non-default) 22.04
-programming environment, you would do the following:
+For example, if you have a code that uses the Gnu compiler environment, FFTW and
+NetCDF parallel libraries and you want to compile in the (non-default and older)
+22.12 programming environment, you would do the following:
 
-First, load the `cpe/23.09` module to switch all the defaults to the versions from
-the 22.04 PE. Then, swap to the GNU compiler environment and load the required library
+First, load the `cpe/22.12` module to switch all the defaults to the versions from
+the 22.12 PE. Then, swap to the GNU compiler environment and load the required library
 modules (FFTW, hdf5-parallel and NetCDF HDF5 parallel). The loaded module list shows they 
-are the versions from the 22.04 PE:
+are the versions from the 22.12 PE:
 
 
 ```bash
-module load cpe/23.09
```

Comment:
is this the right way round? 23.09 is new default so wouldn't need to module load this normally?